src/main/java/nl/tudelft/cse1110/andy/execution/ExecutionFlow.java

```diff
@@ -56,9 +56,9 @@ public static ExecutionFlow build(Context ctx, ResultBuilder result, ResultWrite
 
     private List<ExecutionStep> basicSteps() {
         return Arrays.asList(
-                new SetSecurityManagerStep(),
+                //new SetSecurityManagerStep(),
```

Comment:
Should we disable it? Please, confirm, @martinmladenov

Reply:
Currently the plan is for him to work on this branch and to uncomment these lines when everything is working again. That way we can at least test that the tests are working without having to wait for the security manager to work. Maybe I should have specified 😅